test/test_jit_cuda_fuser.py

```diff
@@ -1488,7 +1488,15 @@ def test_native_layer_norm_bfloat(self):
                 norm_shape = [input_shape[idx] for idx in range(dims - offset, dims)]
                 self._native_layer_norm_helper(input_shape, norm_shape, torch.bfloat16, "cuda", 1e-1)
 
-    def _norm_helper(self, shape, dtype, device, error, is_batch_norm_else_instance_norm, memory_format=torch.contiguous_format):
+    def _norm_helper(self,
+                     shape,
+                     dtype,
+                     device,
+                     error,
+                     is_batch_norm_else_instance_norm,
+                     memory_format=torch.contiguous_format,
+                     *,
```

Comment:
What does this do?

Reply:
This is just to add a keyword only argument for `layer_dtype`, just want to differentiate that one from the rest, since cross-dtype compatibility of layer type and inputs are not well supported in PyTorch.